benchopt/dataset_util.py

```diff
@@ -25,11 +25,11 @@
 def download_libsvm(X_path, y_path, name, replace=False):
     url = ("https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/" +
            NAMES[name] + ".bz2")
-    path = download(url, pjoin(DATA_DIR, name), replace=replace)
+    path = download(url, os.path.join(DATA_DIR, name), replace=replace)
 
     decompressor = BZ2Decompressor()
-    with open(pjoin(path, NAMES[name].split('/')[-1]), "r+b") as f, \
-            open(path, "rb") as g:
+    decomp_path = os.path.join(path, NAMES[name].split('/')[-1])
+    with open(decomp_path, "r+b") as f, open(path, "rb") as g:
```

Comment:
much better!

Reply:
I tend to prefer naming `g->original_file` and `f->temp_file` to be more explicit.